dv/uvm/core_ibex/scripts/render_config_template.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import pathlib3x as pathlib
+from mako.template import Template
+
+from metadata import LockedMetadata
```

Comment:
```suggestion
from metadata import LockedMetadata, RegressionMetadata
```

Reply:
RegressionMetadata isn't used (explicitly at least) within the file, so do we really need this?

Reply:
I agree it's not used explicitly but I get an error locally without it?
```bash
harry@harrycallahan-neptune:~/projects/ibex/dv/uvm/core_ibex$ make core_config
WARNING:ibex_cmd:Rejecting test riscv_bitmanip_full_test, 'rtl_params' specified not compatible with ibex_config
mkdir -p out/build
Generating core configuration file
Traceback (most recent call last):
  File "/home/harry/projects/ibex/dv/uvm/core_ibex/scripts/render_config_template.py", line 35, in <module>
    _main()
  File "/home/harry/projects/ibex/dv/uvm/core_ibex/scripts/render_config_template.py", line 30, in _main
    with LockedMetadata(args.dir_metadata, __file__) as md:
  File "/home/harry/projects/ibex/dv/uvm/core_ibex/scripts/metadata.py", line 448, in __enter__
    self.md = pickle.load(self.handle)
AttributeError: Can't get attribute 'RegressionMetadata' on <module '__main__' from '/home/harry/projects/ibex/dv/uvm/core_ibex/scripts/render_config_template.py'>
make: *** [Makefile:256: out/metadata/core.config.stamp] Error 1

```

Reply:
Hmmm odd, I better add it then :)

Reply:
Also seeing the same issue locally now I have made the change.